fighthealthinsurance/rest_views.py

```diff
@@ -18,90 +126,27 @@ def get(self, request):
 class CheckStorage(APIView):
     def get(self, request):
         es = settings.EXTERNAL_STORAGE
-        with Timeout(2.0) as _timeout_ctx:
-            list = es.listdir("./")
+        with Timeout(2.0):
+            es.listdir("./")
             return Response(status=status.HTTP_204_NO_CONTENT)
         return Response(status=status.HTTP_503_SERVICE_UNAVAILABLE)
 
 
 class CheckMlBackend(APIView):
     def get(self, request):
-        from fighthealthinsurance.model_router import model_router
-
         if model_router.working():
             return Response(status=status.HTTP_204_NO_CONTENT)
-        else:
```

Comment:
Most of these views were converted to the viewsets above, but the logic remains the same.